apps/web/tsconfig.json

```diff
@@ -6,7 +6,8 @@
     "**/*.tsx",
     ".next/types/**/*.ts",
     "../../packages/types/*.d.ts",
-    "../../packages/lib/jwt.ts"
+    "../../packages/lib/jwt.ts",
+    "../../packages/ee/RoleManagement/components/EditMembershipRole.tsx"
```

Comment:
I'm unsure if we need this as we import the whole ee package as dependency. Is it also working without this?
We would have the same issue with the ui package.